go-controller/pkg/cni/types/types.go

```diff
@@ -80,6 +80,15 @@ type NetConf struct {
 	// network mapping in the hosts.
 	PhysicalNetworkName string `json:"physicalNetworkName,omitempty"`
 
+	// Transport describes the transport protocol for east-west traffic.
+	// Valid values are "nooverlay", "geneve", and "evpn".
+	// Defaults to "geneve".
+	Transport string `json:"transport,omitempty"`
```

Comment:
@pliurh headsup this part might be conflicting with https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5780/commits/6698692414971346304c780b56753be68f9b4dcf but its fine, whichever PR lands first, the other one can rebase.